sources/Adapters/picoTracker/filesystem/picoTrackerFileSystem.cpp

```diff
@@ -41,26 +37,35 @@ void picoTrackerPagedDir::GetContent(const char *mask) {
   FsBaseFile entry;
 
   // Insert a parent dir path given that FatFS doesn't provide it
-  subdirIndexes_.push_back(MAGIC_PARENT_DIR_INDEX_MARKER);
+  PathIndex pi;
+  pi.index = 0;
+  pi.type = ParentDirIndex;
```

Comment:
PathIndex pi = {0, ParentDirIndex}

Reply:
brill thanks, I forgot about fixing that after I have finished debugging 👍🏻